### PR TITLE
Titlecaps in Troubleshooting FAQ

### PR DESCRIPTION
**Pull request by kingthorin**
Related to zaproxy/zaproxy#2000.

### CODE REVIEW

**Comment by kingthorin** [comment]
This was submitted via the web UI. I've set DCO pass as per all my recent PRs.

File: site/content/faq/somethings-not-working-what-should-i-do.md [hunk 1]
```diff
@@ -53,17 +53,17 @@ you.
...
+###  Ask on IRC
 
 Many of ZAP contributors hang out on the Libera Chat [#zaproxy](https://web.libera.chat/#zaproxy) channel which you can access via that link or your favourite irc client.
 
-###  Ask on twitter
+###  Ask on Twitter
 
 You can ask [@psiinon](https://twitter.com/psiinon) short questions directly
 on twitter, although whether he's online will be timezone and holiday
 dependant.
```
Comment (ricekot) [inline site/content/faq/somethings-not-working-what-should-i-do.md:64]: Unrelated to title caps; this could be changed to "dependent".

Comment (kingthorin) [reply site/content/faq/somethings-not-working-what-should-i-do.md:64]: Sure

**Comment by kingthorin** [comment]
Tweaked

**Review by ricekot — APPROVED** [review]
Thanks!

**Review by thc202 — APPROVED** [review]

**Pull request merged**